extensions/TheShovel/ShovelUtilsTW.js

```diff
@@ -0,0 +1,258 @@
+console.log("ShovelUtils v1.1")
+var tempImageLoad = {};
```

Comment:
All of these variables need to go underneath `(function(Scratch) {` below so they don't pollute the global scope